beta-20161006.md

```diff
@@ -6,7 +6,7 @@ summary: Additions and changes in CockroachDB version beta-20161006.
 
 ## Oct 6, 2016
 
-Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.  
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to release numerous features currently on our `develop` branch.
```

Comment:
"This week's release does not contain many user-visible changes. This is an artifact of our recent stability efforts, which funneled user-visible changes to a secondary development branch. With the conclusion of the concerted stability effort, we are preparing to include these features in next week's release."